### DISCUSSION TITLE
State Handler not working for colyseus-examples

### DISCUSSION THREAD

**Discussion by rockcs1992** [colyseus/colyseus#776] [Q&A]
Hi,

I'm trying to build something based off [colyseus-examples](https://github.com/colyseus/colyseus-examples), specifically on top of 02-state-handlers. I was able to install and launch the example project, but it looks like 02-state-handlers is not working properly. 

By looking at the code it seems a div should be added whenever a user joins and the user is able to move the div and sync the position with the other users. But that's not happening for
 me. By stepping into the code it seems that this [callback ](https://github.com/colyseus/colyseus-examples/blob/master/src/static/02-state-handler.html#L53C13-L53C68) is never triggered for some reason. I confirmed that the state has been updated on the server side

Could someone verify that? Or I'm not understanding it correctly?

I'm on node 21 and an M1 macbook if that helps 

![Screenshot 2024-10-05 at 1 49 44 PM](https://github.com/user-attachments/assets/b902690c-9e6b-40bf-a697-4400bcf49bea)

**Comment by Cap202209** [comment]
I test the colyseus example from the source here, https://github.com/colyseus/colyseus-examples. and it works correctly.
You may check your project directory again or maybe you can check is there any error log on the browser console.
